spec/jekyll-contentful/data_exporter_spec.rb

```diff
@@ -75,7 +75,7 @@
 
       it 'serializes entries onto data file' do
         allow(File).to receive(:open).and_yield(StringIO.new)
-        expect_any_instance_of(::Jekyll::Contentful::Serializer).to receive(:to_yaml)
+        expect(YAML).to receive(:dump)
```

Comment:
Uhm yeah, this is probably me never having written Ruby tests. I understood the original test would test that the exporter calls the serializer's `to_yaml` method. Since it's no longer doing that, I was testing that it calls `YAML.dump`. Probably I should add a test that it also calls the `serialize` method of the Serializer?